eng/pipelines/runtime-staging-manual.yml

```diff
@@ -0,0 +1,4 @@
+trigger: none
+
+extends:
+    template: runtime-staging-template.yml
```

Comment:
now that we have a template, maybe we can use parameters to turn on/off legs rather than checking based on pipeline name?

Reply:
Like platform!

Reply:
that works for runtime-staging but we still have runtime.yml, so I'd need to move that to template as well which I'm not sure is desired.

Reply:
That one also has a schedule trigger, so it will limit us as well and we will need the template anyway?

Reply:
I think this is a good first step.  I think we can get more custom in a follow up PR.